test/tests/meshgenerators/poly2tri_mesh_generator/poly2tri_from_1d_sbdids.i

```diff
@@ -0,0 +1,38 @@
+[Mesh]
```

Comment:
You shouldn't need `Outputs/exodus=true` anywhere in here. Because you're running with `--mesh-only` only, we never hit Output objects. The mesh only output is separate

Reply:
Does --mesh-only hardcode Exodus as the output format?  What do you do if you need something like Nemesis that doesn't serialize or something like XDA that doesn't lose data?

Reply:
Yes, it does. Typically if you're getting into the non-exodus regime you use split mesh